server/events.go

```diff
@@ -26,6 +26,8 @@ import (
 	"sync/atomic"
 	"time"
 
+	"github.com/nats-io/nuid"
+
```

Comment:
Don't need extra line here.

Reply:
This how go imports does it :( I can remove it but some vet and lints will moan if not like this

Reply:
With VSCode, the std lib imports are separated from the others with 1 blank line, but other imports are grouped without blank line in between. It also orders them alphabetically.